src/interceptors/response.ts

```diff
@@ -62,14 +63,23 @@ export function defaultResponseInterceptor(
       return { ...response, cached: false };
     }
 
-    const config = response.config;
-    const cache = await axios.storage.get(id, config);
-
     // Update other entries before updating himself
-    if (cacheConfig?.update) {
+    if (cacheConfig.update) {
       await updateCache(axios.storage, response, cacheConfig.update);
     }
 
+    if (!isMethodIn(config.method, cacheConfig.methods)) {
+      if (__ACI_DEV__) {
+        axios.debug?.({
+          msg: `Ignored because method (${config.method}) is not in cache.methods (${cacheConfig.methods})`
```

Comment:
```suggestion
          id,
          msg: `Ignored because method (${config.method}) is not in cache.methods (${cacheConfig.methods})`
          data: { config, cacheConfig }
```

Adds more information in case someone needs it.

Reply:
Done!